src/test/kotlin/com/sourcegraph/cody/config/SettingsMigrationTest.kt

```diff
@@ -190,22 +198,20 @@ class SettingsMigrationTest : BasePlatformTestCase() {
   fun `test DeprecatedChatLlmMigration`() {
```

Comment:
That is incorrect I think.
That is migration function, so it specifically migrates data form one old format to another.
It needs to work after your changes.
Then your migration function can pick up state after all existing migrations, and migrate it further.

Reply:
Understood. I reverted those changes and added a new test for my migration.